Automattic/wp-calypso
# Design Picker: Add pattern based virtual themes under a virtual-themes/onboarding flag

Related to https://github.com/Automattic/dotcom-forge/issues/1774

## Proposed Changes

* Display the pattern-based virtual themes in the design picker

## Testing Instructions



* Apply D103329-code patch on your sandbox
* Use the Calypso live link and go to `/setup/site-setup/designSetup?siteSlug=your-site.wordpress.com&flags=virtual-themes/onboarding`
* Make sure that you get the new pattern-based BC3 themes
* Make sure that the theme thumbnails are working
* Click on of them and make sure that the theme preview works for the theme
* Click on the Start button
* Make sure that the pattern is now added to your site.

**Testing the design picker**
 * Notice that we display more categories than before (these categories are because we add the pattern categories in the vTheme response)
 * Click on the categories and notice that vThemes are filtered properly.

## Pre-merge Checklist
